src/main/java/control/FileMenuController.java

```diff
@@ -0,0 +1,150 @@
+                                                                   .size());
+            }
+        }
+    }
+    
+    /**
+     * Add the blocks that were loaded from file to the UI.
+     * @param project the project that was loaded
+     */
+    public void addLoadedBlocks(ScriptingProject project) {
+        log.info("Adding loaded blocks");
+        for (int i = 0; i < project.getCameraTimelines().size();i++) {
+            CameraTimeline timeline = project.getCameraTimelines().get(i);
+            int amountShots = timeline.getShots().size();
+            for (int j = 0; j < amountShots;j++) {
+                CameraShot shot = timeline.getShots().get(j);
+                addCameraShotForLoad(i, shot);
+            }
+        }
+    }
+    
+    /**
+     * Add a camera shot that is loaded from file.
+     * @param cameraIndex the index of the camera timeline to use
+     * @param shot the shot to add
+     */
+    private void addCameraShotForLoad(int cameraIndex, CameraShot shot) {
+        CameraShotBlock shotBlock = new CameraShotBlock(shot, cameraIndex, 
+                                                       controllerManager.getRootPane()
+                                                                        .getRootCenterArea(),
+                                                       controllerManager.getTimelineControl()
+                                                                        ::shotChangedHandler);
+        controllerManager.setActiveShotBlock(shotBlock);
+        controllerManager.getTimelineControl().getCameraShotBlocks().add(shotBlock);
+    }
+    
+    public void newProject() {
+        new NewProjectModalView(controllerManager.getRootPane(), this::createProject);
+    }
+    
+    public void newProject(MouseEvent event) {
+        newProject();
+    }
+    
+    /**
+     * Creates a ScriptingProject from the given event.
+     * @param event the NewProjectCreationEvent given by the creation modal
+     */
+    private void createProject(NewProjectCreationEvent event) {
+        ScriptingProject project = new ScriptingProject(event.getDescription(),
+                                                        event.getSecondsPerCount());
+        project.setDirectorTimeline(new DirectorTimeline(event.getDirectorTimelineDescription(),
+                                                         null));
+        project.setCameras(event.getCameras());
+        project.setCameraTimelines(event.getTimelines());
+        project.getDirectorTimeline().setProject(project);
+        for (CameraTimeline timeline : project.getCameraTimelines()) {
+            timeline.setProject(project);
+        }
+        
+        controllerManager.setScriptingProject(project);
+        RootCenterArea area = new RootCenterArea(controllerManager.getRootPane(),
+                                                 event.getTimelines().size(),
+                                                 false);
```

Comment:
I think it's better to put the parameters all below each other in situations like this, rather than 1 on 1 line and 2 on the other line